modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -723,16 +723,7 @@ definition {
 			userPassword = "${userPassword}");
 	}
 
-	macro deleteObjectEntryViaAPI {
-		JSONObject.deleteObjectEntry(
-			fieldName = "${fieldName}",
-			objectName = "${objectName}",
-			userEmailAddress = "${userEmailAddress}",
-			userPassword = "${userPassword}",
-			value = "${value}");
-	}
-
-	macro deleteObjectEntryViaUI {
+	macro deleteCustomObjectViaUI {
```

Comment:
It looks like the macro was renamed here and deleteObjectEntryViaAPI was sorted to pass SF. It would be better to update the name of the macro in the commit that added it, instead of fixing it in a different commit.